src/main/java/org/prebid/server/analytics/AnalyticsReporter.java

```diff
@@ -11,7 +13,7 @@
      * <p>
      * Implementation note: this method is executed on Vert.x event loop thread so it must never use blocking API.
      */
-    <T> void processEvent(T event);
+    <T> Future<T> processEvent(T event);
```

Comment:
Why we need `Future<T>` ?
As i see we use result as a marker of successful or failed operation. Would be better to use `Future<Void>` ?